simpleSequential/test_floorDisivion.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from floorDivision import floorDivision
+
+class TestFloorDivision(unittest.TestCase):
```

Comment:
The test definition are good. I would love if you keep the description. But, can you change the number a little bit. Because we are trying to test “floor division”, so it it better to have something lest perfect like “10 / (-2)”? Maybe “11 / (-2)”?

Reply:
Please review my old comments.

Reply:
Please review this commit. Let give a try on the input 11 and -2 instead of 10 and -2